backend/s3/s3.go

```diff
@@ -4419,6 +4419,10 @@ func (f *Fs) copyMultipart(ctx context.Context, copyReq *s3.CopyObjectInput, dst
 
 	srcSize := src.bytes
 	partSize := int64(f.opt.CopyCutoff)
+	if f.opt.CopyCutoff <= 0 { //making sure we don't divide by zero
```

Comment:
I don't think this bit of code can be called any more can it? So we don't need a check here.

Reply:
yes, I was debugging into this line and make a change so it was included in this commit. Actually, we don't need it anymore